apps/fluent-tester/ios/Podfile

```diff
@@ -24,7 +24,8 @@ use_test_app! do |target|
   target.app do
     platform :ios, '14.0'
 
-    # There is a bug where autolinking isn't working, do specify these manually.
+    # There is a bug where autolinking isn't working, so specify these manually.
+    pod 'FRNAppearanceAdditions', :path => '../../../packages/experimental/AppearanceAdditions/FRNAppearanceAdditions.podspec'
     pod 'FRNFontMetrics', :path => '../../../packages/experimental/NativeFontMetrics/FRNFontMetrics.podspec'
```

Comment:
That bug might be fixed... so worth a shot to see if we can remove these.

Reply:
Nope, tried removing these and they are extremely not in the workspace. There might be more we need to do to get autolinking working, but I'm gonna punt on investigating that for right now.